components/inputs/test/input-time-range.visual-diff.js

```diff
@@ -235,14 +235,14 @@ describe('d2l-input-time-range', () => {
 	describe('skeleton', () => {
 
 		before(async() => {
-			await page.reload();
-			await page.$eval('#labelled', (elem) => {
-				const timeElem = elem.shadowRoot.querySelector('d2l-input-time');
-				return new Promise((resolve) => {
-					elem.updateComplete.then(() => {
-						timeElem.addEventListener('d2l-input-time-hidden-content-width-change', resolve);
-					});
-				});
+			await page.$eval('#hidden-labels', async(elem) => {
```

Comment:
These skeleton tests were sometimes not run which seemed to be because `resolve` was never called. This seems to work without the `page.reload` and with just resizing these cases back to their original sizes instead.